frontend/__tests__/services/personal-information-service.server.test.ts

```diff
@@ -0,0 +1,82 @@
+import { HttpResponse } from 'msw';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+
+import { getPersonalInformationService } from '~/services/personal-information-service.server';
+
+global.fetch = vi.fn();
+
+vi.mock('~/utils/logging.server', () => ({
+  getLogger: vi.fn().mockReturnValue({
+    info: vi.fn(),
+    error: vi.fn(),
+    debug: vi.fn(),
+    audit: vi.fn(),
+    trace: vi.fn(),
+  }),
+}));
+
+vi.mock('~/utils/env.server', () => ({
+  getEnv: vi.fn().mockReturnValue({
+    INTEROP_CCT_API_BASE_URI: 'https://api.example.com',
+    INTEROP_CCT_API_SUBSCRIPTION_KEY: '00000000000000000000000000000000',
+    INTEROP_CCT_API_COMMUNITY: 'CDCP',
+    ENGLISH_LANGUAGE_CODE: 1033,
+    FRENCH_LANGUAGE_CODE: 1036,
```

Comment:
These mocked environment variables aren't used by `personal-information-service` and can be removed.